npm WARN discord-bot@1.0.43 No description
npm WARN discord-bot@1.0.43 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 615 packages in 3.338s

75 packages are looking for funding
  run `npm fund` for details

found 14 vulnerabilities (2 low, 2 moderate, 10 high)
  run `npm audit fix` to fix them, or `npm audit` for details

> discord-bot@1.0.43 start /home/minecraft/multicraft/servers/server193021
> node index.js

Loaded cat.js
Loaded coin.js
Loaded dog.js
Loaded meme.js
Loaded rps.js
Loaded type.js
Loaded all commands in category fun
Loaded donate.js
Loaded help.js
Loaded info.js
Loaded leaderboard.js
Loaded online.js
Loaded ping.js
Loaded prefix.js
Loaded rank.js
Loaded serverinfo.js
Loaded stats.js
Loaded support.js
Loaded uptime.js
Loaded userinfo.js
Loaded all commands in category info
Loaded clear.js
Loaded config.js
Loaded debug.js
Loaded end.js
Loaded giveaway.js
Loaded lockdown.js
Loaded reroll.js
Loaded unlock.js
Loaded all commands in category management
Loaded avatar.js
Loaded emotes.js
Loaded members.js
Loaded nick.js
Loaded all commands in category misc
Loaded ban.js
Loaded clearpunishments.js
Loaded kick.js
Loaded mute.js
Loaded punishments.js
Loaded tempban.js
Loaded tempmute.js
Loaded unban.js
Loaded unmute.js
Loaded warn.js
Loaded all commands in category moderation
Loaded autoplay.js
Loaded join.js
Loaded nowplaying.js
Loaded pause.js
Loaded play.js
Loaded queue.js
Loaded repeat.js
Loaded resume.js
Loaded seek.js
Loaded shuffle.js
Loaded skip.js
Loaded status.js
Loaded stop.js
Loaded volume.js
Loaded all commands in category music
[DisTube] Downloaded youtube-dl 2021.06.06
Connected to the Quabot database.
 
GENERAL INFORMATION: 
Client: QuaBot#6758
Channels: 451
Servers: 15
Users: 24936
Version: 1.0.5.2
 
Command `play` was used.
[33mWARNING:[0m ytdl-core is out of date! Update with "npm install ytdl-core@latest".
 
You are running Quabot, this project is made by do#8888. It is not allowed to distribute these files punblicly without the creator's consent.
 
Command `help` was used.
Command `help` was used.
Command `coin` was used.
Command `type` was used.
Command `play` was used.
Command `skip` was used.
Command `play` was used.
Command `skip` was used.
Command `play` was used.
Command `play` was used.
Command `skip` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `uptime` was used.
Command `uptime` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `lockdown` was used.
Command `info` was used.
Command `lockdown` was used.
(node:43) UnhandledPromiseRejectionWarning: Error: val is not a non-empty string or a valid number. val=undefined
    at module.exports (/home/minecraft/multicraft/servers/server193021/node_modules/ms/index.js:34:9)
    at /home/minecraft/multicraft/servers/server193021/commands/management/lockdown.js:47:64
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:43) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Command `giveaway` was used.
Command `giveaway` was used.
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 6)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 7)
Command `play` was used.
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 8)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 9)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 10)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 11)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 12)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 13)
Command `skip` was used.
Command `play` was used.
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 14)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 15)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 16)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 17)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 18)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 19)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 20)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 21)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 22)
Command `giveaway` was used.
Command `giveaway` was used.
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 23)
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 24)
Command `giveaway` was used.
Command `giveaway` was used.
Command `play` was used.
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 25)
Command `skip` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 26)
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 27)
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at runNextTicks (internal/process/task_queues.js:66:3)
    at processImmediate (internal/timers.js:429:9)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 28)
Command `play` was used.
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 29)
Command `debug` was used.
Command `skip` was used.
Command `help` was used.
Command `status` was used.
Command `play` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `debug` was used.
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 30)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 31)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 32)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 33)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 34)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 35)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 36)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 37)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 38)
Command `giveaway` was used.
(node:43) UnhandledPromiseRejectionWarning: ReferenceError: Discord is not defined
    at Object.execute (/home/minecraft/multicraft/servers/server193021/commands/management/giveaway.js:131:31)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:43) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 39)
Command `info` was used.
